apisix/plugins/cors.lua

```diff
@@ -227,14 +227,15 @@ end
 
 
 function _M.rewrite(conf, ctx)
+    ctx["raw-request-origin"] = core.request.header(ctx, "Origin")
```

Comment:
```suggestion
    ctx.original_request_origin = core.request.header(ctx, "Origin")
```
would be better.

Please also add a comment about this change.

Reply:
OK